src/ImageSharp/Memory/Allocators/ArrayPoolMemoryAllocator.CommonFactoryMethods.cs

```diff
@@ -8,69 +8,44 @@ namespace SixLabors.ImageSharp.Memory
     /// </summary>
     public partial class ArrayPoolMemoryAllocator
     {
-        /// <summary>
-        /// The default value for: maximum size of pooled arrays in bytes.
-        /// Currently set to 24MB, which is equivalent to 8 megapixels of raw RGBA32 data.
-        /// </summary>
-        internal const int DefaultMaxPooledBufferSizeInBytes = 24 * 1024 * 1024;
-
-        /// <summary>
-        /// The value for: The threshold to pool arrays in <see cref="largeArrayPool"/> which has less buckets for memory safety.
-        /// </summary>
-        private const int DefaultBufferSelectorThresholdInBytes = 8 * 1024 * 1024;
-
-        /// <summary>
-        /// The default bucket count for <see cref="largeArrayPool"/>.
-        /// </summary>
-        private const int DefaultLargePoolBucketCount = 6;
-
-        /// <summary>
-        /// The default bucket count for <see cref="normalArrayPool"/>.
-        /// </summary>
-        private const int DefaultNormalPoolBucketCount = 16;
-
-        // TODO: This value should be determined by benchmarking
-        private const int DefaultBufferCapacityInBytes = int.MaxValue / 4;
-
         /// <summary>
         /// This is the default. Should be good for most use cases.
         /// </summary>
         /// <returns>The memory manager.</returns>
         public static ArrayPoolMemoryAllocator CreateDefault()
-        {
-            return new ArrayPoolMemoryAllocator(
-                DefaultMaxPooledBufferSizeInBytes,
-                DefaultBufferSelectorThresholdInBytes,
-                DefaultLargePoolBucketCount,
-                DefaultNormalPoolBucketCount,
-                DefaultBufferCapacityInBytes);
-        }
+            => new ArrayPoolMemoryAllocator(maxPooledArrayLengthInBytes: DefaultMaxArrayLengthInBytes);
 
         /// <summary>
-        /// For environments with very limited memory capabilities, only small buffers like image rows are pooled.
```

Comment:
I want to remove these factory methods unless we are sure we can provide useful and accurate defaults

Reply:
We should even consider renaming the whole class TBH, I'd postpone figuring out these details until the very end.